terraform/gcp/README.md

```diff
@@ -0,0 +1,259 @@
+unzip nomad.zip
+sudo mv nomad /usr/local/bin
+nomad --version
+```
+
+### Consul
+
+Download the latest version of [Consul](https://www.consul.io/) from HashiCorp's website by copying and pasting this snippet in the terminal:
+
+```console
+curl "https://releases.hashicorp.com/consul/1.8.0/consul_1.8.0_linux_amd64.zip" -o consul.zip
+unzip consul.zip
+sudo mv consul /usr/local/bin
+consul --version
+```
+
+### Vault
+
+Download the latest version of [Vault](https://www.vaultproject.io/) from HashiCorp's website by copying and pasting this snippet in the terminal:
+
+```console
+curl "https://releases.hashicorp.com/vault/1.4.3/vault_1.4.3_linux_amd64.zip" -o vault.zip
+unzip vault.zip
+sudo mv vault /usr/local/bin
+vault --version
+```
+
+### Packer
+
+Download the latest version of [Packer](https://www.packer.io/) from HashiCorp's website by copying and pasting this snippet in the terminal:
+
+```console
+curl "https://releases.hashicorp.com/packer/1.6.0/packer_1.6.0_linux_amd64.zip" -o packer.zip
+unzip packer.zip
+sudo mv packer /usr/local/bin
+packer --version
+```
+
+### Terraform
+
+Download the latest version of [Terraform](https://www.terraform.io/) from HashiCorp's website by copying and pasting this snippet in the terminal:
+
+```console
+curl "https://releases.hashicorp.com/terraform/0.12.28/terraform_0.12.28_linux_amd64.zip" -o terraform.zip
+unzip terraform.zip
+sudo mv terraform /usr/local/bin
+terraform --version
+```
+
+### Install the GCP SDK Command Line Tools
+
+> **Note**: if you are using the free [Google Cloud Shell](https://cloud.google.com/shell) VM, you will already have `gcloud` installed. So, you can safley skip this step.
+
+To install the GCP SDK Command Line Tools, follow the installation instructions for your specific operating system: 
+
+* [Linux](https://cloud.google.com/sdk/docs/downloads-interactive#linux)
+* [MacOS](https://cloud.google.com/sdk/docs/downloads-interactive#mac)
+* [Windows](https://cloud.google.com/sdk/docs/downloads-interactive#windows)
+
+#### Initialize the SDK
+
+To perform common setup tasks like authorizing with GCP using `gcloud`, if you haven't already done so, run the following command:
+
+```console
+gcloud init
+```
+
+## Create a New Project
+
+Create a new GCP project with the following command:
+
+```console
+gcloud projects create your-new-project-name
+```
+
+Now export the project name as the `GOOGLE_PROJECT` environment variable:
+
+```console
+export GOOGLE_PROJECT="your-new-project-name"
+```
+
+And then set your `gcloud` config to use that project:
+
+```console
+gcloud config set project $GOOGLE_PROJECT
+```
+
+### Link Billing Account to Project
+
+Next, let's link a billing account to that project. To determine what billing accounts are available, run the following command:
+
+```console
+gcloud alpha billing accounts list
+```
+
+Then set the billing account ID `GOOGLE_BILLING_ACCOUNT` environment variable:
+
+```console
+export GOOGLE_BILLING_ACCOUNT="XXXXXXX"
```

Comment:
Something like 
"Replace the value XXXXXX with the ACCOUNT_ID for the billing account you want to use."